html/admin/html/form.php

```diff
@@ -17,6 +17,7 @@
 <div>
 <table class="form-table">
 	<tbody>
+	<th><a href="http://help.smaily.com/en/support/solutions/articles/16000062943-create-api-user">How to create API credentials?</a></th>
```

Comment:
This is not translated, and a bit hard to notice. Please move this right after `API Password` input field like we have in other integrations (see https://github.com/sendsmaily/smaily-magento-extension/blob/master/assets/general_settings.png for reference).

Reply:
I was thinking about where to put it, keeping it consistent with other extensions is a good idea.
Wrote it down that I should update translations but forgot anyway, added in commit.